Jenkinsfile

```diff
@@ -74,9 +75,36 @@ pipeline{
                     else
                         cp $ML_RPM .
                     fi
+                if [-z ${env.ML_CONVERTERS}]; then
```

Comment:
@itsshivaverma Let me know if you have any ideas about how to get converters into the docker image in Jenkins, I basically copied the logic for the RPM installation and altered the Path for the `scp` step.

Reply:
Build just passed for the current iteration of the Jenkinsfile.